### ISSUE TITLE
Patient lookup results list does not include dob and address

### ISSUE THREAD

**Issue by wlthomson** [openmsupply/mobile#2712]
## Is your feature request related to a problem? Please describe.

The patient lookup results list only contains first and last name (@craigdrown).

Maybe confusing with prescriber? Patient result list includes DOB (but no address).

## Describe the solution you'd like

Possibly leave as is? Adding address would be tricky, not much real estate to work with.

Should probably shrink the date format to `DD/MM/YYYY`, but even then will be tricky to fit address info.

## Implementation

TO DO.

## Describe alternatives you've considered

N/A.

## Additional context

@craigdrown Feedback from dispensary demo (17/04/20).

**Comment by joshxg** [comment]
Just thinking of alternatives/out the box:

What if after searching, the modal closed and the records showed in the normal patient list. Still can't see ALL the info, but you could click the "edit"/"profile" button to view?

**Comment by wlthomson** [comment]
@joshxg I think in order to do that (without a pretty sizable refactor), you'd have to add all the result records to the local db, which would increase the amount of records store in multiple places (and might be a bit confusing in general).

I think the feedback I'm getting indicates that the modal solution is probably going to be a short term one, as the general concensus seems to be that the search functionality needs to be "combined" in some way with local search (which I agree with, but am not sure how would work in practice).

My thinking would be that the dispensing page has an optional "advanced search", which brings up the search form (maybe as a side/bottom modal, or full screen modal) and adds a checkbox (or mobile equivalent) for including remote results. Then when the patient is selected, if the patient is non-local, he/she is added as an extra action dispatched on opening the prescribing wizard.

If I'm honest, I think the reason I didn't go with this sort of approach wasn't taken from the beginning was mainly time constraints. I think designing this well would require a decent amount of thought and possibly a lot of refactoring.

To get back to the issue at hand though, I think including the DOB is probably sufficient, as to me it seems a bit overkill to show address (just my opinion, but I think if a user sees two patients with the same name and DOB but with a different address, they'll probably assume its the same patient with two different addresses!)

TLDR; I think (at least for now) first name, last name and DOB is probably sufficient to provide enough usability to distinguish between patients?
